tests/tests_ecg_findpeaks.py

```diff
@@ -23,6 +24,23 @@ def _read_csv_column(csv_name, column):
     csv_data = pd.read_csv(csv_path, header=None)
     return csv_data[column].to_numpy()
 
+def test_ecg_findpeaks_all_methods_handle_empty_input():
+    METHODS = ["neurokit", "pantompkins", "nabian", "gamboa", 
+               "slopesumfunction", "wqrs", "hamilton", "christov",
+               "engzee", "manikandan", "elgendi", "kalidas", 
+               "martinez", "rodrigues"]
+    
+    failed_methods = []
+    for method in METHODS:
+        try:
+            method_func = _ecg_findpeaks_findmethod(method)
+            _ = method_func(np.zeros(12*240), sampling_rate=240)
+        except Exception:
+            failed_methods.append(method)
+            continue
+    if failed_methods:
+        raise Exception(f"Failed methods: {failed_methods}")
```

Comment:
This also works but [parametrizing the test function](https://docs.pytest.org/en/stable/how-to/parametrize.html) could give a more detailed error message e.g.:
https://github.com/neuropsychology/NeuroKit/blob/781ebd6fe5d6546f2231fe0a3b8cdc78b3c4ac8c/tests/tests_rsp.py#L381-L389

Reply:
fixed in last commit